server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java

```diff
@@ -363,52 +363,15 @@ public void deleteUser(String userId) throws ControlServiceException {
         userService.removeUserById(userId);
     }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see
-     * org.kaaproject.kaa.server.control.service.ControlService#getTenantAdmins
-     * ()
-     */
-    @Override
-    public List<TenantAdminDto> getTenantAdmins() throws ControlServiceException {
-        return userService.findAllTenantAdmins();
-    }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see
-     * org.kaaproject.kaa.server.control.service.ControlService#getTenantAdmin
-     * (java.lang.String)
-     */
-    @Override
-    public TenantAdminDto getTenantAdmin(String tenantId) throws ControlServiceException {
-        return userService.findTenantAdminById(tenantId);
-    }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see
-     * org.kaaproject.kaa.server.control.service.ControlService#editTenantAdmin
-     * (org.kaaproject.kaa.common.dto.TenantAdminDto)
-     */
     @Override
-    public TenantAdminDto editTenantAdmin(TenantAdminDto tenantAdmin) throws ControlServiceException {
-        return userService.saveTenantAdmin(tenantAdmin);
+    public List<UserDto> findAllTenantAdminsByTenantId(String tenantId) throws ControlServiceException {
```

Comment:
Please make sure that you have corresponding REST API call and method in AdminClient